src/components/DateInput.svelte

```diff
@@ -1,40 +1,34 @@
 <script lang="ts">
-import { MDCTextField } from '@material/textfield'
-import { onMount } from 'svelte'
 
 export let value = ''
 export let disabled = false
 export let autofocus = false
-
-let element = {}
-let mdcTextField = {}
-
-onMount(() => {
-  mdcTextField = new MDCTextField(element)
-  return () => mdcTextField.destroy()
-})
+export let padding = '12px'
+export let width = '200px'
 
 const focus = (node) => autofocus && node.focus()
 </script>
 
 <style>
-.input-label {
-  width: 228px;
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+.custom-field {
+  font-size: 14px;
+  position: relative;
+  border-top: 20px solid transparent;
 }
 
-.date-input {
-  padding-right: 60px;
+.custom-field input {
+  border-radius: 8px;
+  border: 1px solid gray;
+  padding: var(--field-padding) var(--field-padding) var(--field-padding) var(--field-padding);
```

Comment:
If these are all the same, is there a reason not to use the shorter syntax?
```suggestion
  padding: var(--field-padding);
```